BarcodeHero.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |spec|
     # Root Specification
     spec.name     = 'BarcodeHero'
-    spec.version  = '0.2.5'
+    spec.version  = '0.2.6'
```

Comment:
Like you mentioned, let's at least bump to 0.3. When we're in beta releases for SemVer, we should treat the minor as volatile/breaking changes.